feature-libs/asm/components/asm-main-ui/asm-main-ui.component.ts

```diff
@@ -165,49 +164,54 @@ export class AsmMainUiComponent implements OnInit, OnDestroy {
           }
         })
     );
-    this.checkIsEmulateFromDeepLinkAndNavigate();
     this.subscribeForDeeplink();
   }
 
-  /**
-   * If emulate customer, navigate to home page
-   * */
-  protected checkIsEmulateFromDeepLinkAndNavigate(): void {
```

Comment:
We want to keep it here when input as below, we'll navigate to home page(i.e, route "/assisted-service/emulate" to "/"

http://localhost:4200/electronics-spa/en/USD/assisted-service/emulate?customerId=703a67be-c3c5-4b61-ab71-ed0e5a964f4f

https://help.sap.com/docs/SAP_COMMERCE_CLOUD_PUBLIC_CLOUD/e1391e5265574bfbb56ca4c0573ba1dc/c61971cf2c974fa3bb59fa7f3cca9878.html?q=ASM%20Deep%20Link

Ah, Maybe the name confliction has some misleading "asmEnableService.isEmulatedByDeepLink()" is used to indicates whether the URL contains "assisted-service/emulate?" but  "asmComponentService.isEmulatedByDeepLink()" is used to indicate the asm module status.

I just changed the "this.asmEnableService?.isEmulatedByDeepLink" => this.asmEnableService?.isEmulateInURL() in my branch.